extensions/wikia/WikiFactory/tests/WikiFactoryTest.php

```diff
@@ -155,6 +163,39 @@ public function testGetLocalEnvURLDataProvider() {
 		];
 	}
 
+	public function testPrepareUrlToParseDataProvider() {
+		return [
+			[
+				'http://www.community-name.wikia.com',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "http://www.community-name.wikia.com" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)